MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl

```diff
@@ -1054,6 +1054,408 @@ resource configservice_label portal_config_manager_portal_labels_language {
   ]
 }
 
+resource configservice_label portal_config_manager_portal_labels_title_import_export {
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_TITLE_IMPORT_EXPORT'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Import/Export labels'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Import/eksport mærkater'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label portal_config_manager_portal_labels_description_import_export {
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_DESCRIPTION_IMPORT_EXPORT'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Export the labels in your chosen language as an Excel sheet using the Export button. Any changes you make in the Excel file can be imported using the Import button.'
```

Comment:
```suggestion
      default_translation = 'Export all {{language}} labels as an Excel sheet using the Export button. Any changes you make in the Excel file can be imported using the Import button.'
```

Reply:
this is the general description, no language available here